reactor-core/src/main/java/reactor/core/scheduler/ReactorBlockHoundIntegration.java

```diff
@@ -50,5 +50,7 @@ public void applyTo(BlockHound.Builder builder) {
         // For now, let's not add a separate integration, but rather let's define the class name manually
         // ContextRegistry reads files as part of the Service Loader aspect. If class is initialized in a non-blocking thread, BlockHound would complain
         builder.allowBlockingCallsInside("reactor.core.publisher.ContextPropagation", "<clinit>");
+        builder.allowBlockingCallsInside("java.util.concurrent.FutureTask",
```

Comment:
use `FutureTask.class.getName()` instead of `"java.util.concurrent.FutureTask"`

Reply:
Update with class reference